--- docker-image/v1.15/debian-kinesis/Gemfile
+++ docker-image/v1.15/debian-kinesis/Gemfile
@@ -8 +8 @@
-gem "rexml", "~> 3.2.5"
+gem "rexml", "3.3.9"
--- docker-image/v1.15/debian-loggly/Gemfile
+++ docker-image/v1.15/debian-loggly/Gemfile
@@ -8 +8 @@
-gem "rexml", "~> 3.2.5"
+gem "rexml", "3.3.9"
--- docker-image/v1.16/debian-elasticsearch6/Gemfile
+++ docker-image/v1.16/debian-elasticsearch6/Gemfile
@@ -8 +8 @@
-gem "rexml", "~> 3.2.5"
+gem "rexml", "3.3.9"
--- docker-image/v1.16/debian-s3/Gemfile
+++ docker-image/v1.16/debian-s3/Gemfile
@@ -8 +8 @@
-gem "rexml", "~> 3.2.5"
+gem "rexml", "3.3.9"
--- docker-image/v1.17/debian-logzio/Gemfile
+++ docker-image/v1.17/debian-logzio/Gemfile
@@ -8 +8 @@
-gem "rexml", "~> 3.2.5"
+gem "rexml", "3.3.9"
--- docker-image/v1.17/debian-s3/Gemfile
+++ docker-image/v1.17/debian-s3/Gemfile
@@ -8 +8 @@
-gem "rexml", "~> 3.2.5"
+gem "rexml", "3.3.9"
--- docker-image/v1.18/debian-graylog/Gemfile
+++ docker-image/v1.18/debian-graylog/Gemfile
@@ -8 +8 @@
-gem "rexml", "~> 3.2.5"
+gem "rexml", "3.3.9"
--- docker-image/v1.18/debian-s3/Gemfile
+++ docker-image/v1.18/debian-s3/Gemfile
@@ -8 +8 @@
-gem "rexml", "~> 3.2.5"
+gem "rexml", "3.3.9"
